### PR TITLE
First version.

### PR DESCRIPTION
**Pull request by vext01**
Tiny kernel module and test program to enable `RDPMC` from userspace.

OK?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+obj-m += user_rdpmc.o
+
+all: user_rdpmc.ko test_rdpmc
+
+user_rdpmc.ko: user_rdpmc.c
+	 make -C /lib/modules/$(shell uname -r)/build/ M=$(PWD) modules
+
+test_rdpmc: test_rdpmc.c
+	${CC} ${CFLAGS} -std=gnu99 -o test_rdpmc test_rdpmc.c
```
Comment (ltratt) [inline Makefile]: Use `-O2`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+The user_rdpmc Linux Kernel Module
+==================================
+
+A simple Linux kernel module which, when loaded, allows ring 3 (userspace)
+programs to use the `RDPMC` assembler instruction by flipping the `PMC` flag in
+the `CR4` register.
+
+This module only makes sense on an X86 machine.
```
Comment (ltratt) [inline README.md]: Lowe-case `x`?

**Comment by ltratt** [comment]
One minor comment, otherwise looks good.

**Comment by vext01** [comment]
Pushed a fix. Squash?

**Pull request merged**